lib/puppet/parser/functions/new.rb

```diff
@@ -864,7 +866,7 @@
 | b | The data is in base64 encoding, padding as required by base64 strict is added by default
 | u | The data is in URL safe base64 encoding
 | B | The data is in base64 strict encoding
-| s | The data is a puppet string used verbatim as a binary byte sequence
+| s | The data is a puppet string seen as a binary sequence of bytes with ignored encoding
```

Comment:
I don't think that wording is correct either. The string, when represented as bytes, does have an encoding, even if that encoding is unknown. It can't be ignored. Here, it's apparently up to Ruby to turn the string into a sequence of bytes. How does it do that?

Let's assume we would implement this in Java where a string uses 16-bit characters. Would that mean that you'd get twice as many bytes using this method?